recipe/meta.yaml

```diff
@@ -1,36 +1,47 @@
-{% set version = "3.2.3" %}
+{% set version = "3.2.4.post1" %}
 
 package:
   name: scs
   version: {{ version }}
 
 source:
   - url: https://github.com/bodono/scs-python/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 7592805810fb6da09955dc45322b2d376bca993928c972f472f1a5352821ddcd
+    sha256: bb509507d857906eb95d17d5824138e48e7c75d1a692d0aa7256d899c7e703c5
   # include submodule (not in github tarball due to dear-github/dear-github#214)
   - git_url: https://github.com/cvxgrp/scs.git
-    git_rev: ac6840a3b3264950e6c300264cbf3937e0bcc6c5
-    folder: scs
+    git_rev: 9024b8ccc1bba6ee797440fb22354cadb9c81839
+    folder: scs_source
 
 build:
   number: 0
   script:
-    - export BLAS_LAPACK_LIB_PATHS="$PREFIX/lib"    # [unix]
-    - export BLAS_LAPACK_LIBS="lapack:cblas:blas"   # [unix]
-    - {{ PYTHON }} -m pip install . -vv
+    # Copy cross-compile meson file hack from scipy feedstock:
+    # See https://github.com/conda-forge/scipy-feedstock/blob/9401aa4ee5d9a22c2c9d219e5fb6dbd580f30207/recipe/build.sh#L10
+    - echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt
```

Comment:
This isn't a huge issue and I didn't want to hold things up further, but this should really have at least a `# [unix]` selector, or even better: `# [build_platform != target_platform]`. IIRC the cross-file is only created in that case, and we shouldn't write garbage in there that might end up confusing meson (the reason why it works is that we're not specifying `--cross-file=...` in the native case, so the incomplete config there doesn't get picked up).